src/Surfnet/StepupSelfService/SelfServiceBundle/Tests/Resources/test.crt

```diff
@@ -0,0 +1,18 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Consider adding a new certificate that is not expiring in 3 years. But keeping this one is fine too!

Reply:
Good suggestion!